keys/src/address.rs

```diff
@@ -269,4 +309,71 @@ mod tests {
 		assert_eq!(address, "SVCbBs6FvPYxJrYoJc4TdCe47QNCgmTabv".into());
 		assert_eq!(address.to_string(), "SVCbBs6FvPYxJrYoJc4TdCe47QNCgmTabv".to_owned());
 	}
+
+	#[test]
+	fn test_from_to_cashaddress() {
+		let cashaddresses = vec![
+			"bitcoincash:qzxqqt9lh4feptf0mplnk58gnajfepzwcq9f2rxk55",
+			"bitcoincash:qr6m7j9njldwwzlg9v7v53unlr4jkmx6eylep8ekg2",
+			"bitcoincash:pq4ql3ph6738xuv2cycduvkpu4rdwqge5q2uxdfg6f",
+		];
+		let expected = vec![
+			Address {
+				prefix: 0,
+				t_addr_prefix: 0,
+				hash: vec![140, 0, 44, 191, 189, 83, 144, 173, 47, 216, 127, 59, 80, 232, 159, 100, 156, 132, 78, 192].as_slice().into(),
```

Comment:
Should be sufficient to just `[140, 0, 44, 191, 189, 83, 144, 173, 47, 216, 127, 59, 80, 232, 159, 100, 156, 132, 78, 192].into()` as AddressHash is H160 which is wrapper over `[u8; 20]`.